CHANGES.rst

```diff
@@ -1,6 +1,15 @@
 Release notes
 =============
 
+1.17.1 (2023-01-05)
+-------------------
+
+bug: Adding the use of ItemAdapter to prevent assumptions of item nature (#358)
+bug: Fix Slack dependency name issue (#367)
+chore: Change the Ubuntu version on workflow settings (#373)
+docs: Adding PeriodicExecutionTimeMonitor to the batteries docs (#368)
+misc: Fix compatibility issues with jsonschema>=4 (#364)
```

Comment:
**suggestion** Add the links to the PRs and/or issues related to each of these new features as we already have in all other latest versions changelogs.